src/elements/content-explorer/stories/RenameDialog.stories.js

```diff
@@ -0,0 +1,53 @@
+// @flow
+import * as React from 'react';
+import { useArgs } from '@storybook/preview-api';
+
+import PrimaryButton from '../../../components/primary-button/PrimaryButton';
+import { addRootElement } from '../../../utils/storybook';
+
+import RenameDialog from '../RenameDialog';
+
+// need to import this into the story because it's usually in ContentExplorer
+import '../../common/modal.scss';
```

Comment:
do we still need this since we are using blueprint?

Reply:
nope, ill remove it